.github/workflows/deploy.yml

```diff
@@ -99,11 +99,10 @@ jobs:
             clean \
             html \
             slides \
+            linkcheck \
```

Comment:
To add to the fun there is a broken URL on cylc-flow (which we failed to spot) and that causes deployment to fail (and rightly so).

It's not worth fiddling the cylc-flow 8.0a2 tag (and I don't think we would do that anyway) so I removed `linkcheck` from the build to get the deployment running.

If linkcheck becomes problematic in the future we may need to find other ways to cope with it, but putting `linkcheck` back in for now.